### PR TITLE
Show note text content

### PR DESCRIPTION
**Pull request by josephktcheung**
This PR allows user to define optional note text shown on the note view.

simulator_screenshot_FAFDE9B0-D862-4C43-B8FD-8ECBE74AD235

### CODE REVIEW

**Comment by josephktcheung** [comment]
@aure @wtholliday please review thanks

File: Sources/PianoRoll/PianoRollNoteView.swift [hunk 1]
```diff
@@ -99,8 +99,13 @@ struct PianoRollNoteView: View {
 
         // Main note body.
         ZStack(alignment: .trailing) {
-            Rectangle()
-                .foregroundColor(color.opacity((hovering || offset != .zero || lengthOffset != 0) ? 1.0 : 0.8))
+            ZStack(alignment: .leading) {
+                Rectangle()
+                    .foregroundColor(color.opacity((hovering || offset != .zero || lengthOffset != 0) ? 1.0 : 0.8))
+                Text(note.text ?? "")
```
Comment (wtholliday) [inline Sources/PianoRoll/PianoRollNoteView.swift:109]: What's the behavior of this zstack if the text is longer than the rectangle?

Comment (josephktcheung) [reply Sources/PianoRoll/PianoRollNoteView.swift:109]: Text will be truncated

https://user-images.githubusercontent.com/4270232/198055419-2a3d2b2e-9dc3-44b0-bd4e-e2188098631d.mp4

Comment (josephktcheung) [reply Sources/PianoRoll/PianoRollNoteView.swift:109]: SwiftUI is smart enough to determine if text should be truncated based on the frame limit we set in https://github.com/AudioKit/PianoRoll/pull/7/files#diff-e2fedbc9a6fe669091b2ad9ea5f5e70164631e6de19aacf75ab318969b725707R116-R117

**Comment by aure** [comment]
If you want to resolve the conflicts, I'll merge this.

**Comment by josephktcheung** [comment]
> If you want to resolve the conflicts, I'll merge this.

@aure conflicts resolved.

**Pull request merged**